api/scripts/wazuh-apid.py

```diff
@@ -68,43 +69,61 @@ def start(foreground, root, config_file):
         print(f"Starting API in background")
         pyDaemonModule.pyDaemon()
 
+    cluster_config = read_config()
+    configuration.api_conf = configuration.read_api_config(config_file=args.config_file)
```

Comment:
Not sure if this is correct. The `api_conf` is a mutable object. It should be modified, not replaced. What you are replacing here is the reference to the object.